submissions/Vlada-khom/js-dom/main.js

```diff
@@ -0,0 +1,130 @@
+const data = [
+  {
+    name: "Abyssinian",
+    id: 'b48bec34-a43c-447d-a309-2cc3978e72b9',
```

Comment:
I think it will be more appropriate if you change the ids to prime numbers. (1, 2, 3, 4....)